mypy/build.py

```diff
@@ -413,12 +422,13 @@ def correct_rel_imp(imp: Union[ImportFrom, ImportAll]) -> str:
 
             return new_id
 
-        res = []  # type: List[Tuple[str, int]]
+        res = []  # type: List[Tuple[int, str, int]]
```

Comment:
Yes, and that's what happens later, at the call site (line 1280 below once I've pushed a new version).